demo_scripts/AKSDeployment/README.md

```diff
@@ -0,0 +1,314 @@
+Welcome to this tutorial where we will take you step by step in creating an AKS Application with a custom domain that is secured via https. This tutorial assumes you are logged into Azure CLI already and have selected a subscription to use with the CLI. It also assumes that you have helm installed (Instructions ca be found here https://helm.sh/docs/intro/install/). If you have not done this already. Press b and hit ctl c to exit the program.
+
+To Login to Az CLI and select a subscription 
+'az login' followed by 'az account list --output table' and 'az account set --subscription "name of subscription to use"'
+
+To Install Az CLI
+If you need to install Azure CLI run the following command - curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+
+
+Assuming the pre requisites are met press enter to proceed
+
+## Define Command Line Variables 
+Most of these variables should be set to a smart default. However, if you want to change them
+press b and run the command export VARIABLE_NAME="new variable value"
+
+```
+echo $RESOURCE_GROUP_NAME
```

Comment:
We need to standardize on common environment variables. This allows improved reuse of the content across different scenarios.  No change requested for this doc... just a note to self that we need a coding standards doc